P/protobuf-c/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "protobuf_c"
+version = v"1.4.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/protobuf-c/protobuf-c/releases/download/v$(version)/protobuf-c-$(version).tar.gz", "4cc4facd508172f3e0a4d3a8736225d472418aee35b4ad053384b137b220339f")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd protobuf-c-1.4.1/
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
```

Comment:
Haven't tried, but this _should_ make it find the right `protoc`
```suggestion
./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} PROTOC="$(which protoc)"
```